tests/e2e/koperator_suite_test.go

```diff
@@ -55,8 +55,9 @@ var _ = BeforeSuite(func() {
 
 var _ = When("Testing e2e test altogether", Ordered, func() {
 	//testInstall()
-	//testProduceConsumeInternal()
-	testUninstallZookeeperCluster()
-	testUninstallKafkaCluster()
-	testUninstall()
+	testProduceConsumeExternal("kafka-controller")
+	// testProduceConsumeInternal()
+	// testUninstallZookeeperCluster()
+	// testUninstallKafkaCluster()
+  // testUninstall()
 })
```

Comment:
**Question**: What is the reason for commenting out the internal producer/consumer test?
My impression is that `testProduceConsumeInternal` and `testProduceConsumeExternal` are leaf level tests - they contain everything else from the rest of the cases and bits of extra stuff except for the actual internal and external producer/consumer tests which are unique to those two - so I would think to have those 2 enabled when we want to run `make test-e2e`.

Reply:
After David PR we can remove all of the commenting out for other test cases. We need the deploying kafkacluster and zookeepercluster to make them complete. Until that we cannot do complex tests. I commented out because I could test only this way by deploying kafkacluster and zookeeper by hand and run e2e test